### PR TITLE
修改Readme描述方式

### PR DESCRIPTION
**Pull request by niauah**

### CODE REVIEW

**Review by sih4sing5hong5 — APPROVED** [review]
Aray~

File: README.md [hunk 1]
```diff
@@ -44,143 +44,81 @@ services:
...
+- `S3_REGION`: The name of the S3 region (eg. `eu-west-1`). This may be optional depending on your storage vendor.
+- `S3_BUCKET` (required): The bucket name.
+- `S3_ACCESS_KEY_ID` (required): The S3 Access Key ID.
+- `S3_SECRET_ACCESS_KEY` (required): The S3 Secret Access Key.
 
-## Environment Variables
+### Backup Schedule
 
-### S3_ENDPOINT_URL
+- `SCHEDULE`: The backup schedule specified in a string following [crontab syntax](https://www.man7.org/linux/man-pages/man5/crontab.5.html) where the five fields are minute, hour, day of month, month and day of week. If set to a blank string, the script will perform a manual backup and exit. The default value is a blank string.
```
Comment (sih4sing5hong5) [inline README.md]: > ... the script will perform a **manual** backup and exit. The default value is a blank string. ...

寫` perform a backup and exit`，mài `manual`，敢會較清楚？

Comment (niauah) [reply README.md]: 656c4ef

Comment (sih4sing5hong5) [reply README.md]: Aray~

**Pull request merged**